pimcore/lib/Pimcore/Bundle/CoreBundle/Command/InternalNewsletterDocumentSendCommand.php

```diff
@@ -16,11 +16,13 @@
 
 use Pimcore\Console\AbstractCommand;
 use Pimcore\Document\Newsletter\AddressSourceAdapterInterface;
+use Pimcore\Document\Newsletter\AddressSourceAdapterFactoryInterface;
 use Pimcore\Logger;
 use Pimcore\Model;
 use Pimcore\Tool\Newsletter;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Intl\Exception\RuntimeException;
```

Comment:
This was accidentally i suppose?

Reply:
no, thats the exception thrown in line 71. Better idea for a different Exception Type? Maybe ClassNotFound or InvalidArgument?

Reply:
`\RuntimeException` or `\InvalidArgumentException` but in any case none of Symfony's intl component.